spec/index.html

```diff
@@ -538,72 +538,70 @@ <h2>RDF Graphs</h2>
   <section id="section-triples">
     <h3>Triples</h3>
 
-    <p>An <dfn data-local-lt="triple">RDF triple</dfn> is a 3-tuple that is defined recursively as follows:</p>
+    <p>An <dfn data-local-lt="triple">RDF triple</dfn> (usually called "triple")
+      is a 3-tuple (|s|, |p|, |o|) where:</p>
+
+    <ul>
+      <li>|s| is an <a>IRI</a> or a <a>blank node</a>,</li>
+      <li>|p| is an <a>IRI</a>, and</li>
+      <li>|o| is an <a>IRI</a>, a <a>blank node</a>, a <a>literal</a>,
+        or a <a>triple term</a>.</li>
+    </ul>
+
+    <p>A <dfn>triple term</dfn> is a 3-tuple that is defined recursively as follows:</p>
 
     <ul>
       <li>
         If |s| is an <a>IRI</a> or a <a>blank node</a>,
         |p| is an <a>IRI</a>, and
         |o| is an <a>IRI</a>, a <a>blank node</a>, or a <a>literal</a>,
-        then (|s|, |p|, |o|) is an RDF triple.
+        then (|s|, |p|, |o|) is a triple term.
       </li>
 
       <li>
-        If |t| and <var>t'</var> are <a>RDF triples</a>,
-        |s| is an <a>IRI</a> or a <a>blank node</a>,
+        If |s| is an <a>IRI</a> or a <a>blank node</a>,
         |p| is an <a>IRI</a>, and
-        |o| is an <a>IRI</a>, a <a>blank node</a>, or a <a>literal</a>,
-        then (|t|, |p|, |o|), (|s|, |p|, |t|), and (|t|, |p|, <var>t'</var>) are RDF triples.
+        |o| is a <a>triple term</a>,
+        then (|s|, |p|, |o|) is a triple term.
```

Comment:
Again, this describes an RDF triple, but in this case containing <var>t`</var>, <var>t`</var> is a triple term, but <var>t</var> is an RDF triple.

Reply:
Based on the same rationale as in [my reply to your other comment](https://github.com/w3c/rdf-concepts/pull/78/files#r1495450008), I think this should be 'triple term' here.